content/cif-connector/src/main/content/jcr_root/libs/commerce/gui/components/authoring/editor/pagepreview/renderconditions/isproductpage/isproductpage.jsp

```diff
@@ -0,0 +1,57 @@
+<%--
+
+    Copyright 2019 Adobe. All rights reserved.
+    This file is licensed to you under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License. You may obtain a copy
+    of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software distributed under
+    the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+    OF ANY KIND, either express or implied. See the License for the specific language
+    governing permissions and limitations under the License.
+
+--%><%
+%><%@include file="/libs/granite/ui/global.jsp" %><%
```

Comment:
Why do you use a JSP here? A servlet or possibly HTL are more appropriate and allow unit testing the Java code.